### PR TITLE
Feature/work section with about me section

### PR DESCRIPTION
**Pull request by 0xMALVEE**
added work section and about me section.
displaying all skills and past works.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -88,5 +88,208 @@ <h2>
       </p>
     </div>
   </section>
+
+  <section class="work-section">
+    <header class="work-heading">
+      <h2>My Recent Works</h2>
+      <hr class="line">
+    </header>
+    <article class="work-card-container">
```
Comment (NahnahAJ) [inline index.html:97]: - [ ] I would suggest that you style the main container of the projects using a `display:grid` so a to meet the Project Requirements. I have attached a screenshot of the PR below for your reference.
- Project R

Comment (0xMALVEE) [reply index.html:97]: done just pushed now

File: index.html [hunk 2]
```diff
@@ -88,5 +88,208 @@ <h2>
...
+            </ul>
+            <button class="view-project-btn">
+              See Project
+            </button>
+          </div>
+        </div>
+      </div>
+
+      <div class="sub-projects">
+        <div class="sub-project-container">
```
Comment (NahnahAJ) [inline index.html:121]: - [ ] For the content of each of the sub-projects, according to the project requirements you need to use flexbox to lay out each of them :+1:  
- [ ] Kindly do this for each of the sub-projects. :100: 
- 
image

File: index.html [hunk 3]
```diff
@@ -88,5 +88,208 @@ <h2>
...
+          <button class="view-project-btn">
+            <span>See Project</span>
+          </button>
+        </div>
+      </div>
+
+    </article>
+  </section>
+
+  <section class="aboutme-section">
```
Comment (NahnahAJ) [inline index.html:237]: - [ ] Kindly make use of flexbox to align the content of the `aboutme-section ` as stated in the project requirement so as to meet all the requirements :100: 
- Project Requirement
image

File: index.html [hunk 4]
```diff
@@ -88,5 +88,208 @@ <h2>
...
+              <li> <a href="#">bootstrap</a> </li>
+              <li> <a href="#">Ruby</a> </li>
+            </ul>
+          </div>
+        </div>
+        <div class="btn-container">
+          <button class="view-project-btn">
+            <span>See Project</span>
+          </button>
+        </div>
```
Comment (NahnahAJ) [inline index.html:231]: - [ ] Apart from the first project, all the other projects currently don't show and except the buttons. Kindly fix this as well :+1: I have attached a video below for your reference.
- Video Reference
[Screencast from 02-16-2023 06:24:41 AM.webm](https://user-images.githubusercontent.com/102699958/219285769-4bac1c23-8df6-49d6-b559-07cb77c6043c.webm)

File: index.html [hunk 5]
```diff
@@ -88,5 +88,208 @@ <h2>
...
+    </article>
+  </section>
+
+  <section class="aboutme-section">
+    <header class="aboutme-heading">
+      <h2 class="title">About Me</h2>
+      <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="resume-btn">Get My Resume</button>
+    </header>
+    <hr>
```
Comment (NahnahAJ) [inline index.html:243]: - [ ] I noticed that you are missing a background image here. So kindly fix this as well. Please try to adhere to the Figma design guidelines as much as possible :100: 
- Your Design
image
- The Figma Design
image

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @0xMALVEE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] Good use of GitHub flow :heavy_check_mark: 
- [ ] No linter errors :heavy_check_mark: 
- [ ] Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (0xMALVEE) [reply index.html:243]: I do have a bg image in my local server I can see it idk why you can't see it. download the files correctly I think

Comment (0xMALVEE) [reply index.html:243]: I did put in my readme.md file on how to run my site locally but I think you haven't seen it and given me the wrong info. If you use a live server 80% of the things you mention work. Please read everything carefully.

File: index.html (hunk 4 above)
Comment (0xMALVEE) [reply index.html:231]: you should have read my readme.md and used a live-server to run my code locally. That's not my code problem. 
```
npm install -g live-server
live-server
```

Comment (Shedrack-Sunday) [reply index.html:231]: Hello @0xMALVEE , i can confirm its there. reviewing your project already

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @0xMALVEE 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**